core-services/prow/02_config/operator-framework/operator-controller/_prowconfig.yaml

```diff
@@ -0,0 +1,18 @@
+tide:
+  merge_method:
+    operator-framework/operator-controller: squash
+  queries:
+  - includedBranches:
+    - main
+    labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
```

Comment:
Do we have jira integration?

Reply:
That plugin is not enabled [here](https://github.com/openshift/release/pull/63229/files#diff-4aa3feb876b1b68160c859277e3b82d045694d5103aed44d6fa833094d5d68c9R13).  
I didn't figure that we needed to ignore/omit the label, since these are "do not merge if present" label lists.